### ISSUE TITLE
Don't store images in memory

### ISSUE THREAD

**Issue by oparoz** [nextcloud/gallery#242]
@rullzer mentioned we did this for some of the images. It would be good if he could point me in the right direction again, so that I can have a look :)

**Comment by rullzer** [comment]
right... let me assign myself so I either look it up for you or create a pr...

**Comment by rullzer** [comment]
It is basically happening at https://github.com/nextcloud/gallery/blob/master/preview/preview.php#L68

But the code is mixed with the path for thumbnails as well. Probabaly it would make sense to solve this together with https://github.com/nextcloud/gallery/issues/245

**Comment by MorrisJobke** [comment]
I guess this is nothing for 12, right?
